packages/builder-rsbuild/src/preview/virtual-module-mapping.ts

```diff
@@ -21,12 +22,12 @@ export const getVirtualModules = async (options: Options) => {
   const virtualModules: Record<string, string> = {}
   const cwd = process.cwd()
   const workingDir = options.cache
-    ? path.resolve(
-        process.cwd(),
-        // TODO: This is a hard code cache dir, as Rspack doesn't support virtual modules now.
-        // Remove this when Rspack supports virtual modules.
-        './node_modules/.cache/storybook/storybook-rsbuild-builder',
-      )
+    ? // TODO: This is a hard code cache dir, as Rspack doesn't support virtual modules now.
+      // Remove this when Rspack supports virtual modules.
+      (findCacheDirectory({
+        name: 'storybook-rsbuild-builder',
```

Comment:
Is it on purpose to change the cache dir from `.cache/storybook/storybook-rsbuild-builder` to `.cache/storybook-rsbuild-builder` here?

Reply:
I think it's fine to place it in either location. I can't recall why I initially put it inside `./storybook`—maybe it was just an unconscious choice. I'm glad to see we're now aligned with Storybook. Thank you.

Reply:
Thank you!